packages/firebase_messaging/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.0.13
+
+* Fix error invoking `configure` with `onBackgroundMessage` on iOS.
```

Comment:
@ericmartineau Please add some information here on what the fix was. Eg: Only call onBackgroundMessage on Android.

Reply:
apologies if this is written somewhere, but what's the procedure once I've fixed and pushed?  Should I hit the "Resolve conversation" button?  Request a re-review?